oak-core/src/main/java/org/apache/jackrabbit/oak/namepath/impl/GlobalNameMapper.java

```diff
@@ -67,10 +68,15 @@ protected static boolean isHiddenName(String name) {
     protected static boolean isExpandedName(String name) {
         if (name.startsWith("{")) {
             int brace = name.indexOf('}', 1);
-            return brace != -1 && name.substring(1, brace).indexOf(':') != -1;
-        } else {
-            return false;
+            if (brace != -1) {
+                String namespace = name.substring(1, brace);
+                // the empty namespace and "internal" are valid as well, otherwise it always contains a colon (as it is a URI)
```

Comment:
As most people do not understand the difference between URI and "relative reference" we may want to refer to Section 3 of RFC 3986.